server/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -345,9 +344,8 @@ static boolean executeBulkItemRequest(
 
         final IndexShard primary = context.getPrimary();
         final long version = context.getRequestToExecute().version();
-        final boolean isDelete = context.getRequestToExecute().opType() == DocWriteRequest.OpType.DELETE;
         final Engine.Result result;
-        if (isDelete) {
+        if (context.getRequestToExecute().opType() == DocWriteRequest.OpType.DELETE) {
```

Comment:
Sorry this is still somewhat unrelated, forgot to inline this in the previous PR already. No nered for the variable now that we only use this once.